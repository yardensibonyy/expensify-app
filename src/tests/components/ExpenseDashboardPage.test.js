import React from 'react';
import { shallow } from 'enzyme';
import ExpanseDashboardPage from '../../components/ExpenseDashboardPage';

test('should render ExpenseDashboardPage correctly to the screen', () => {
    const wrapper = shallow(<ExpanseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});