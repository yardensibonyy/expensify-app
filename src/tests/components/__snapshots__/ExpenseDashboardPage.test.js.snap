// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correctly to the screen 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;

